exercises/bowling/src/test/java/BowlingTest.java

```diff
@@ -0,0 +1,268 @@
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+
+public class BowlingTest {
+    private BowlingGame game;
+    
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Before
+    public void setup() {
+        game = new BowlingGame();
+    }
+    
+    private void PlayGame(int[] rolls)
```

Comment:
typically only constructors start with a capital letter. Instead of `PlayGame` it should be `playGame`